prime-dotnet-webapi/Controllers/EnrolleesController.cs

```diff
@@ -77,7 +77,7 @@ public async Task<ActionResult<IEnumerable<Enrollee>>> GetEnrollees([FromQuery]E
         [ProducesResponseType(typeof(ApiResultResponse<Enrollee>), StatusCodes.Status200OK)]
         public async Task<ActionResult<Enrollee>> GetEnrolleeById(int enrolleeId)
         {
-            var enrollee = await _enrolleeService.GetEnrolleeAsync(enrolleeId);
+            var enrollee = await _enrolleeService.GetEnrolleeAsync(enrolleeId, User.IsAdmin());
```

Comment:
this should probably be User.HasAdminView()

Reply:
Thanks @james-hollinger updated